spec/app/models/metasploit_data_models/automatic_exploitation/match_result_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe MetasploitDataModels::AutomaticExploitation::MatchResult do
```

Comment:
This is missing the following spec sections:

* Associations
* CONSTANTS
* Mass assignment security
* Validations
* Scopes